examples/basic_button.rb

```diff
@@ -14,13 +14,12 @@
 
 UI.window_on_closing(main_window) do
   puts 'Bye Bye'
-  UI.control_destroy(main_window)
```

Comment:
I am curious, why do you not call UI.control_destroy anymore? I automated calling destroy on window and all its children in Glimmer DSL for LibUI. Does that mean I don't need to call destroy explicitly anymore when using `LibUI.uninit`?

Reply:
Thank you for answering my question.